src/constant/index.js

```diff
@@ -291,3 +291,74 @@ export const CROSS_SELL_PRODUCT_IDS_MAP = IS_TESTNET
         'likenft1r8mce5h96nvedtm7rh9va5mda6fgysqfprlxftuef09s8kfjt2jsqeyyjq',
       ],
     };
+
+export const AFFILIATION_CHANNELS = [
```

Comment:
Any reason we are not using a map here?
Also maybe naming it more explitcitly like `AFFILIATION_CHANNEL_LEGACY_STRINGS` would be better